libpf/trace.go

```diff
@@ -100,3 +100,59 @@ type Trace struct {
 	Hash         TraceHash
 	CustomLabels map[String]String
 }
+
+// EbpfTrace represents a stack trace from Ebpf code.
+type EbpfTrace struct {
```

Comment:
This type defintion might be better placed in package `support` and maybe we could generate the non-mapp[] part in some automated way. Changes to `Comm`, `ProcessName` or their order might otherwise cause significant issues.